test/pkg/integration/context/context.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package context
+
+import (
+	"context"
+
+	"github.com/gitpod-io/gitpod/test/pkg/integration"
+)
+
+type componentKey struct{}
+
+type workspaceKey struct{}
+
+func GetComponentAPI(ctx context.Context) *integration.ComponentAPI {
```

Comment:
/nit 

Maybe 'MustGet*' variants to make the calling code a touch more explicit?